.github/workflows/release-step-3.yml

```diff
@@ -151,6 +151,7 @@ jobs:
         run: ./scripts/docker-release/push_docker.sh "${{ env.RELEASE_VERSION }}"
 
   publish-aws-lambda:
+    if: false
```

Comment:
@reakaleek is this expected?

Reply:
No it's not. Thank you. Probably some leftover state of my fork.

```suggestion
```